client/web/src/search/home/SearchPage.tsx

```diff
@@ -64,7 +66,12 @@ export const SearchPage: React.FunctionComponent<React.PropsWithChildren<SearchP
         <div className={classNames('d-flex flex-column align-items-center px-3', styles.searchPage)}>
             <BrandLogo className={styles.logo} isLightTheme={props.isLightTheme} variant="logo" />
             {props.isSourcegraphDotCom && (
-                <div className="text-muted text-center mt-3">Search millions of open source repositories</div>
+                <div className="d-flex flex-row">
+                    <div className={classNames('text-muted text-center mt-3 mr-2 pr-2', styles.subtitle)}>Search millions of open source repositories</div>
```

Comment:
suggestion: you can use the existing utility instead of adding a new class.

```suggestion
                    <div className={classNames('text-muted text-center mt-3 mr-2 pr-2 border-right')}>Search millions of open source repositories</div>
```

Reply:
✅  Thank you, @limitedmage!